tests/Command/PsyshCommandIntegrationTest.php

```diff
@@ -61,6 +61,16 @@ public function testScopeVariables()
         );
     }
 
+    public function testContainerInstance()
+    {
+        $container = $this->shell->getScopeVariable('container');
+        if (in_array(getenv('SYMFONY_VERSION'), ['~3.4.0', '~4.0.0'], true)) {
+            $this->assertInstanceOf('Symfony\Component\DependencyInjection\Container', $container);
```

Comment:
I would rather use `Symfony\Component\DependencyInjection\Container::class` here (which even if the class doesn't exists results in the same string)